OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -25,27 +25,39 @@
     <Reference Include="Eluant, Version=1.0.5229.27703, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\thirdparty\Eluant.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
OK I got the main idea. When running nunit, all dlls related to the testing project must be placed in the working directory of the dll it is testing (in this case MainFolder/OpenRA.Test). Since OpenRA.Mods.Common is located in a different directory (say bin/Debug) NUnit is unable to locate it.

There are a couple of options for making this work:

1- Change the Mods.Common project so that the built dll is placed in the root directory.
2- Change the Test project and keep everything inside him private (like before).

If anyone has any other approaches, feel free to suggest.